methods.py

```diff
@@ -409,8 +409,7 @@ def mySubProcess(cmdline, env):
             "shell": False,
             "env": env,
         }
-        if sys.version_info >= (3, 7, 0):
-            popen_args["text"] = True
+        popen_args["text"] = True
```

Comment:
Seems like it ran with your version of Python that's newer and removed important compatibility code for Python 3.6?

Reply:
Ah shoot, can't do much about that until 3.6 is no longer the minimum version. I guess that specific rule can be excluded though, will look into later